qa/build.xml

```diff
@@ -29,11 +29,12 @@
   <!-- you should not need to configure anything below here -->
   <!--
         This is a redhat vpn server. for non-vpn use, mirror it locally or copy the drivers by hand.
-        wget \-\-no-parent -m http://www.qa.jboss.com/jdbc-drivers/
-        wget \-\-no-parent -m http://www.qa.jboss.com/jdbc-drivers/jConnect-6_0/classes
+        wget \-\-no-parent -m http://www-qa-jboss.hosts.mwqe.eng.bos.redhat.com/jdbc-drivers/
+        wget \-\-no-parent -m http://www-qa-jboss.hosts.mwqe.eng.bos.redhat.com/jdbc-drivers/jConnect-6_0/classes
         ant -Ddriver.url=file:///path/to/www.qa.jboss.com/jdbc-drivers get.drivers
     -->
-  <property name="driver.url" value="http://www.qa.jboss.com/jdbc-drivers/"/>
+  <property name="driver.url" value="https://www-qa-jboss.hosts.mwqe.eng.bos.redhat.com/jdbc-drivers/"/>
+  <property name="driver.oracle.url" value="https://svn.devel.redhat.com/repos/jboss-qa/hibernate-properties-and-drivers/jdbc/EAP/7.4.0/"/>
```

Comment:
I will review this PR after the CI runs finish but this line depends on a private repo managed by Red Hat whereas narayana is a public community project. This means that the QA tests would only be able to be ran behind a firewall. Is the driver available from any public repository?

Reply:
That line should be removed. @jmfinelli if it is necessary to do this then you could consider if getting the URL to use from an environment variable in the job config is possible?

Reply:
My bad, I added the two servers without double-checking they are private servers. Problem is that http://www.qa.jboss.com/ looks to be down and the only alternative I found was www-qa-jboss.hosts.mwqe.eng.bos.redhat.com (which I realise now it is a private server)

Reply:
@jmfinelli if it is anywhere here: https://search.maven.org/search?q=ojdbc if so you could trying putting the dependency under https://github.com/jbosstm/narayana/blob/master/qa/pom.xml#L200

Reply:
I'll try to implement your suggestion @tomjenkinson. As this modules runs an ant script to execute tests, I thought it was worth to give a shot to the (already-in-place) logic to fetch drivers. As you and @mmusgrov suggested, the server hosting JDBC drivers should not be behind a VPN.

I'll try to copy the new oracle driver from Maven to the folder where the ant script looks into and, if successful, we might think to apply the same logic to all other drivers.